pkgs/build-support/setup-hooks/auto-patchelf.sh

```diff
@@ -99,11 +108,4 @@ autoPatchelf() {
 #
 # shellcheck disable=SC2016
 # (Expressions don't expand in single quotes, use double quotes for that.)
```

Comment:
This comment refers to code no longer in evidence.

Reply:
Good catch!